client/components/Players/PlayerManager.vue

```diff
@@ -572,6 +572,17 @@ export default Vue.extend({
           ]
         });
       }
+    },
+    togglePictureInPicture(): void {
+      // @ts-expect-error - `pictureInPictureElement` does not exist in relevant types
+      if (document.pictureInPictureElement) {
+        // @ts-expect-error - `exitPictureInPicture` does not exist in relevant types
+        document.exitPictureInPicture();
+        // @ts-expect-error - `pictureInPictureEnabled` does not exist in relevant types
+      } else if (document.pictureInPictureEnabled) {
+        // @ts-expect-error - `togglePictureInPicture` does not exist in relevant types
```

Comment:
Why not use a different method to disable the errors to avoid four duplicate comments?

Reply:
Such as?

You can't currently comment out blocks in TypeScript @Microsoft/TypeScript#19573

And I couldn't find any type libraries like we have for media sessions

Reply:
I think you can create an interface for it with type void

Reply:
I've extended the `document` type to include the relevant functions/expressions. But I'm not a fan of it :/

Reply:
@Camc314 I wasn't aware of the line-specific limitation, I agree the new method is less readable than your previous code :sweat_smile: I think it's fine to revert that change.

Reply:
Ah no worries, I've reverted the change.